cli/flox/doc/manifest.toml.md

```diff
@@ -253,27 +253,73 @@ DB_URL = "http://localhost:2000"
 SERVER_PORT = "3000"
 ```
 
+## `[hook]`
+
+The `[hook]` section of the manifest allows you to specify scripts to be invoked by a
+**bash** shell immediately upon activating an environment for the *first time only*.
+Subsequent activations, such as the "in-place" activations performed by way of
+"in-place" invocations performed in subshell "dotfiles" will *NOT* re-run these scripts.
+
+Hook scripts inherit environment variables set in the `[vars]` section,
+and variables set by the script will in turn be inherited by the `[profile]`
+scripts described below. Any stdout generated by the scripts in this section
+will be redirected to stderr to avoid it being interpreted as code for
+"in-place" activations.
```

Comment:
```suggestion                                                                                                                                                                                                                                                                   
Hook scripts inherit environment variables set in the `[vars]` section,                                                                                                                                                                                                         
and variables set by the script will in turn be inherited by                                                                                                                                                                                                                    
the `[profile]` scripts described below.                                                                                                                                                                                                                                        
Any output written to `stdout` in a hook script is redirected to `stderr`                                                                                                                                                                                                       
to avoid it being mixed with the output of profile section scripts                                                                                                                                                                                                              
that must be able to write to `stdout` for "in-place" activations.                                                                                                                                                                                                              
```